src/main/java/io/mystore/shipping/action/ShippingToOrderSkuItemAction.java

```diff
@@ -36,9 +36,9 @@ public Effect<Empty> onOrderCreated(ShippingEntity.OrderCreated orderCreated) {
         .toList();
 
     var result = CompletableFuture.allOf(results.toArray(new CompletableFuture[results.size()]))
-        .thenApply(reply -> effects().reply(Empty.getDefaultInstance()));
+        .thenApply(reply -> Empty.getDefaultInstance());
 
-    return effects().asyncEffect(result);
+    return effects().asyncReply(result);
```

Comment:
It's not needed to map to `Effect<Empty>`. We can return a `CompletionStage<Empty>` using `asyncReply`.